TTS/tts/layers/losses.py

```diff
@@ -308,6 +312,9 @@ def __init__(self, c, ga_sigma=0.4):
         # pylint: disable=not-callable
         self.criterion_st = BCELossMasked(pos_weight=torch.tensor(self.stopnet_pos_weight)) if c.stopnet else None
 
+        # For dev pruposes only
```

Comment:
This may be deleted later but it's a good sanity check to compare values with the original paper.